### PR TITLE
docs(typo) Patch a typo in the "Creating a new example" page

### PR DESCRIPTION
**Pull request by swederik**
Do I need to do anything other than edit the Markdown for a docs change? I am assuming the CI will build the docs.

Thanks!

### CODE REVIEW

File: Documentation/content/docs/develop_example.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ To list the examples you can run the following command:
 $ npm run example
 ```
 
-Then base on the list of examples you get, you can run a given one by providing its name as argument like follow:
+Then, based on the list of examples you get, you can run a given one by providing its name as argument like follow:
```
Comment (aronhelser) [inline Documentation/content/docs/develop_example.md]: Can you change "like follow" to "like the following"? Looks good! CI will build the docs, like you suspect.

**Comment by swederik** [comment]
There you are!

**Comment by aronhelser** [comment]
Thanks!

**Pull request merged**